packages/rich-text/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentful/field-editor-rich-text",
-  "version": "1.1.16",
+  "version": "2.0.0-next.43",
```

Comment:
We should be getting v2.0.0 when it's merged to master, correct? even though the current `2.0.0-next.43` is in the `next` npm channel?

Reply:
Looks like we are not using lerna instead of semantic-release to publish. Not sure how to do a dry run to check ☝🏻

Reply:
Running `lerna version --no-private --conventional-commits --no-push --no-git-tag-version --allow-branch rebase-next` resulted in `2.0.0-next.44` -_-

Reply:
I think I may have to release v2.0.0 manually and then merge. The merge will likely result in v2.0.1 but that's Ok.